src/DropdownFilter.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.dropdownfilter.DropdownFilter" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Dropdown filter</name>
+    <description>Filter Mendix list views using a dropdown</description>
+    <icon>
+        iVBORw0KGgoAAAANSUhEUgAAAA0AAAAPCAYAAAA/I0V3AAAAAXNSR0IArs4c6QAAAZ9JREFUKBWNkr9rwkAUx19+GM1voyg4umSTTM4dpFMpurk4V2mXjoVCCx0q3YVi/4UOFUqHCjp1cWzBUqpIZ0WXaE1KTPouFZGo0IPjvXf3/XzfHXdUPp8/HAwGqV6vB5ZlAQ4Wp0bTtMLzPEiS5MiybMdiMRujJQjCF0tRVKrVatUdx/Gh+XwOs5kNo9E3MEwIIRkURQFZloCYVCrFY+IKuVyuXKvV6qZpIjAD05zCZDIFluV8QNM0UFUVqtWLU+wGbKPRuMtkMkfpdBrG4zGKJ+iuoNUUOI6HeDwOiUQColGV+APqb2mSuK7b1nW9jPdAIQehEAs0TWEnFsLhMB6TgVLp4AzjM9H7ULfb7Xue1zYMoxyJRHyQYf4MRFGAYnH/Eu/+2Gw2P1YQSWzb7mNoZ7PZE9KNYSgfLhT2rrHjQ6fTeSe6XUPHjfOnpxdPFMWbZDJpBIX+8QKLn+v1cDh8Xa9Jvg0KajbqXZC3VLobBC7sgn6IeLFYzP8N4Xv5HfBr+XAQ3NoJH5sj3wXj1n0q6LKsrzCSvXucb8u1VfgFhT5/n4nGOh0AAAAASUVORK5CYII=
+    </icon>
+    <properties>
+        <property key="entity" type="entity">
+            <caption>List view entity</caption>
+            <category>General</category>
+            <description>Entity whose attributes will be filtered. This entity should match the list view entity</description>
+        </property>
+        <property key="placeholder" type="translatableString">
+            <caption>Placeholder</caption>
+            <category>General</category>
+            <description>Used as filter placeholder when no default filter exists</description>
```

Comment:
`Placeholder showing when no filter is select, only relevant when empty filter is allowed`

Reply:
Done